src/ui/PrintableUri/PrintableUri.ts

```diff
@@ -17,7 +17,9 @@ export interface IPrintableUriOptions extends IResultLinkOptions {
 }
 
 /**
- * The PrintableUri component displays the URI, or path, to access a result.
+ * The PrintableUri component inherits from the ResultLink component and supports all of its options.
```

Comment:
We should create a link in the documentation ( {@link ResultLink} )

@fbeaudoincoveo  Any comments to add on this ?

I think we should